src/patternfly/components/Button/button.scss

```diff
@@ -172,7 +185,7 @@
   --#{$button}--m-plain--m-no-padding--hover__icon--Color: var(--pf-t--global--icon--color--regular);
   --#{$button}--m-plain--m-no-padding--hover--BackgroundColor: transparent;
   --#{$button}--m-plain--m-no-padding--m-clicked--BackgroundColor: transparent;
-  --#{$button}__progress--Color: var(--#{$button}__icon--Color);
+  --#{$button}--m-plain--m-no-padding--after--Inset: calc(#{pf-size-prem(2px)} * -1);
```

Comment:
Can we change the name of `after--Inset` to `border--offset` for all of these vars? It's more semantic so it isn't necessarily tied to the `::after` element and `inset` property.